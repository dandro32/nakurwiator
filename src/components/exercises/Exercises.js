import React from 'react';
import Dashboard from '../shared/dashboard/Dashboard';

const Exercises = () => {
  const cards = [
    {
      title: 'Dodaj nowe ćwiczenie',
      desc: 'Dodaj nowe ćwiczenie do atlasu ćwiczeń',
      href: '/exercises/new',
      hrefTitle: 'Dawej',
      imgSrc: '/img/exercises_new.jpg'
    },
    {
      title: 'Atlas ćwiczeń',
      desc: 'Zobacz jakie masz ćwiczenia, edytuj lub usuwaj',
      href: '/exercises/list',
      hrefTitle: 'To the atlas',
      imgSrc: '/img/exercises_list.jpg'
    }
  ];
  return (<Dashboard cards={cards} />);
};

export default Exercises;