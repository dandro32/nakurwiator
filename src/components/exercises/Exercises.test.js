import React from 'react';
import Exercises from './Exercises';
import TestRenderer from 'react-test-renderer';

jest.mock('../shared/dashboard/Dashboard', () => 'Dashboard component');

describe('Exercises component tests', ()=> {
  let result;

  beforeEach(() => {
    result = TestRenderer.create(
      <Exercises />
    ).toTree();

  });

  it('renders correctly', () => {
    expect(result).toMatchSnapshot();
  });
});