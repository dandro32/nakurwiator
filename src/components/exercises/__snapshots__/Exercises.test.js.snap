// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Exercises component tests renders correctly 1`] = `
Object {
  "instance": null,
  "nodeType": "component",
  "props": Object {},
  "rendered": Object {
    "instance": null,
    "nodeType": "host",
    "props": Object {
      "cards": Array [
        Object {
          "desc": "Dodaj nowe ćwiczenie do atlasu ćwiczeń",
          "href": "/exercises/new",
          "hrefTitle": "Dawej",
          "imgSrc": "/img/exercises_new.jpg",
          "title": "Dodaj nowe ćwiczenie",
        },
        Object {
          "desc": "Zobacz jakie masz ćwiczenia, edytuj lub usuwaj",
          "href": "/exercises/list",
          "hrefTitle": "To the atlas",
          "imgSrc": "/img/exercises_list.jpg",
          "title": "Atlas ćwiczeń",
        },
      ],
    },
    "rendered": Array [],
    "type": "Dashboard component",
  },
  "type": [Function],
}
`;
